src/wavm.cpp

```diff
@@ -185,24 +185,29 @@ ExecutionResult WavmEngine::execute(
     ensureCondition(false, ContractValidationFailure, "Bug in wavm: didn't check bounds before allocation");
   }
 
-  // next set up the host module.
-  // Note: in ewasm, we create a new VM for each call to a module, so we must instantiate a new host module for each of these VMs, this is inefficient, but OK for prototyping.
+  // compile the module (not sure what this does)
```

Comment:
https://github.com/WAVM/WAVM/blob/master/Include/Runtime/Runtime.h#L339

The `IR::Module` is an AST of the WASM and the `Runtime::Module` is the llvm bitcode. This compiles the first to the second.